lib/change-log.js

```diff
@@ -349,7 +349,7 @@ async function track_changes (req) {
   let isComposition = _isCompositionContextPath(req.context.path)
   let entityKey = diff.ID
 
-  if (req.event === "DELETE") {
+  if (cds.transaction(req).context.event === "DELETE") {
```

Comment:
Hi Nikica,

I mentioned this change in the email, please check the email if you have time.

Best Regards,
Wenjun